src/js/extensions/auto-link.js

```diff
@@ -30,6 +30,8 @@ LINK_REGEXP_TEXT =
 
     AutoLink = Extension.extend({
         init: function () {
+            Extension.prototype.init.apply(this, arguments);
```

Comment:
Yeah, I added this just about everywhere for that exact reason.  I realized this because I needed the anchor extension to be able to override `init()` so it could listen for the keyboard shortcut, but I still wanted to make sure init got called in `FormExtension` in case something got added in there in the future (instead of calling straight to `Button.init()` which would skip a level of 'inheritance')